### PR TITLE
Fix response for `updaterelayfee`

### LINKED ISSUE #1995: Improve `updaterelayfee` JSON response
Using `updaterelayfee` with eclair v0.6.2 returns the following:

```json
{
  "4881ea3050bd0d82a571db79984a4c1c432eb77a7349b1afac6198b92d740cca": "RES_SUCCESS(CMD_UPDATE_RELAY_FEE(null,10 msat,150,None),4881ea3050bd0d82a571db79984a4c1c432eb77a7349b1afac6198b92d740cca) (of class fr.acinq.eclair.channel.RES_SUCCESS)"
}
```

We should instead return something like:

```json
{
  "4881ea3050bd0d82a571db79984a4c1c432eb77a7349b1afac6198b92d740cca": {
    "feeBase": 10,
    "feeProportionalMillionths": 150
  }
}
```

Or in case of error:

```json
{
  "4881ea3050bd0d82a571db79984a4c1c432eb77a7349b1afac6198b92d740cca": "error message"
}
```

If possible, we should probably also fix #2001 at the same time.

### PR DESCRIPTION
**Pull request by pm47**
Fixes #1995, which was due to a pattern matching error for the expected response type of `sendToX` helper methods in `EclairImpl`, and had nothing to do with json serialization. Added a few non-reg tests.

In the second commit I also set a basic "ok" json serializer for all default `RES_SUCCESS` messages, but didn't follow https://github.com/ACINQ/eclair/issues/1995#issuecomment-940821678, because we would either completely break backwards compatibility, or create inconsistency with non-default command responses like `RES_GETINFO`, and with other API calls not related to channels. Seemed better to leave it for another PR.

### CODE REVIEW

File: eclair-core/src/test/scala/fr/acinq/eclair/EclairImplSpec.scala [hunk 1]
```diff
@@ -469,6 +471,121 @@ class EclairImplSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike with I
...
+
+    register.expectMsg(Symbol("channels"))
+    register.reply(map)
+
+    val c1 = register.expectMsgType[Register.Forward[CMD_GETINFO]]
+    register.reply(RES_GETINFO(map(c1.channelId), c1.channelId, NORMAL, ChannelCodecsSpec.normal))
+    val c2 = register.expectMsgType[Register.Forward[CMD_GETINFO]]
+    register.reply(RES_GETINFO(map(c2.channelId), c2.channelId, NORMAL, ChannelCodecsSpec.normal))
+    val c3 = register.expectMsgType[Register.Forward[CMD_GETINFO]]
+    register.reply(RES_GETINFO(map(c3.channelId), c3.channelId, NORMAL, ChannelCodecsSpec.normal))
```
Comment (t-bast) [inline eclair-core/src/test/scala/fr/acinq/eclair/EclairImplSpec.scala:496]: You can add below:

```scala
    register.expectNoMessage()
```

File: eclair-core/src/test/scala/fr/acinq/eclair/EclairImplSpec.scala [hunk 2]
```diff
@@ -469,6 +471,121 @@ class EclairImplSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike with I
...
+    import f._
+
+    val eclair = new EclairImpl(kit)
+
+    val a = randomKey().publicKey
+    val b = randomKey().publicKey
+    val a1 = randomBytes32()
+    val a2 = randomBytes32()
+    val b1 = randomBytes32()
+    val map = Map(a1 -> a, a2 -> a, b1 -> b)
```
[the remark this answers is not in the document]
Comment (pm47) [reply eclair-core/src/test/scala/fr/acinq/eclair/EclairImplSpec.scala]: Done with 278a2e5.

File: eclair-core/src/test/scala/fr/acinq/eclair/EclairImplSpec.scala [hunk 3]
```diff
@@ -469,6 +471,121 @@ class EclairImplSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike with I
...
```
[the remark this answers is not in the document]
Comment (pm47) [reply eclair-core/src/test/scala/fr/acinq/eclair/EclairImplSpec.scala]: Done with 278a2e5.

File: eclair-core/src/test/scala/fr/acinq/eclair/EclairImplSpec.scala (hunk 1 above)
Comment (pm47) [reply eclair-core/src/test/scala/fr/acinq/eclair/EclairImplSpec.scala:496]: Done with 278a2e5.

**Review by t-bast — APPROVED** [review]

**Pull request merged**